examples/tpac_2019.html

```diff
@@ -0,0 +1,424 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>TPAC 2019 - New Features</title>
+    <meta charset='utf-8'>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c' class=
+    'remove'></script>
+    <script class='remove'>
+      var respecConfig = {
+        specStatus: "ED",
+        shortName: "payment-request",
+        editors: [{
+          name: "Marcos Caceres",
+        }, {
+          name: "Kagami Sascha Rosylight",
+        }, {
+          name: "Sid Vishnoi",
+        }],
+        github: "https://github.com/w3c/some-API/",
+        testSuiteURI: "https://w3c-test.org/some-API/",
+        implementationReportURI: "https://w3c.github.io/test-results/some-API",
+        mdn: true,
+        format: "markdown",
+        // localBiblio: {
+        //   "credential-management": { aliasOf: "credential-management-1" },
+        //   "css-scoping": { aliasOf: "css-scoping-1" },
+        // },
+      };
+    </script>
+<body>
+
+<section id="abstract">
+  ## New stuff in ReSpec!
+
+  ... we have been busy! 🏋️‍
+
+</section>
+
+
+## Contributor and stats
+
+  - We've had 37 new contributors project.
+  - Supports numerous students - GSoC, Outreachy
+
+
+
+## Now in two flavours
+ReSpec now comes into flavors:
+
+  - 🍏 respec-w3c.js
+    - 327Kb - (93Kb) gzip
```

Comment:
Hmm, if we ship ReSpec with brotli we could even get < 80 KiB of size. Firefox supports it since version 44, maybe time to support the new algorithm. https://tools.paulcalvano.com/compression.php

![image](https://user-images.githubusercontent.com/3396686/66200481-220d4780-e6dc-11e9-8423-59e9f126a771.png)

Reply:
I thought this would be automatically done on the W3C server (once they add support)? We could ask for support if it’s easy to add. I’m not sure what they use.